### ISSUE TITLE
Aggiunta di file con dati giornalieri

### ISSUE THREAD

**Issue by CapMar00** [italia/covid19-opendata-vaccini#118]
Buongiorno, 
il file somministrazioni-vaccini-latest.csv e il corrispettivo somministrazioni-vaccini-latest.json sono ormai corposi e continuano a crescere.
E' possibile affiancare a questi 2 file, dei file che contengano solo i dati giornalieri in modo da agevolare gli script che vanno a interrogarli?
So che nella repository ci sono dei file che raccolgono alcuni dei dati giornalieri, ma nessuno ha il livello di dettaglio del somministrazioni-vaccini-latest.
Grazie

**Comment by RossoDeM** [comment]
> Buongiorno,
> il file somministrazioni-vaccini-latest.csv e il corrispettivo somministrazioni-vaccini-latest.json sono ormai corposi e continuano a crescere.
> E' possibile affiancare a questi 2 file, dei file che contengano solo i dati giornalieri in modo da agevolare gli script che vanno a interrogarli?
> So che nella repository ci sono dei file che raccolgono alcuni dei dati giornalieri, ma nessuno ha il livello di dettaglio del somministrazioni-vaccini-latest.
> Grazie

Concordo. Proprio oggi ha superato il Maximum Size Limit per l'importazione.
Occorreva pensarci con un certo anticipo, visto che era un problema ampiamente prevedibile

**Comment by Ugetto** [comment]
Sarebbe comodo avere un file con solo i giornalieri, ma non è possibile, visto che ogni volta vengono aggionati i dati di molti giorni precedenti. 
Ad esempio nel file del 25 Marzo rispeto al file del 24 Marzo sono state aggiornate le somministrazioni del 4 Gennaio perchè la regione Liguria ha aggiunto 60 somministrazioni prima non dichiarate. Analogamente la regione Lazio ha aggiunto 18 somministrazioni del 6 Gennaio, ancora il Lazio ha aggiunto 10 somministrazioni del 7 Gennaio, etc......
Le modifiche a ritroso sono tantissime ogni volta e coinvolgono tutto il periodo di vaccinazione.
Non so bene perchè ci sono questi continui riallineamenti.

**Comment by CapMar00** [comment]
> Sarebbe comodo avere un file con solo i giornalieri, ma non è possibile, visto che ogni volta vengono aggionati i dati di molti giorni precedenti.
> Ad esempio nel file del 25 Marzo rispeto al file del 24 Marzo sono state aggiornate le somministrazioni del 4 Gennaio perchè la regione Liguria ha aggiunto 60 somministrazioni prima non dichiarate. Analogamente la regione Lazio ha aggiunto 18 somministrazioni del 6 Gennaio, ancora il Lazio ha aggiunto 10 somministrazioni del 7 Gennaio, etc......
> Le modifiche a ritroso sono tantissime ogni volta e coinvolgono tutto il periodo di vaccinazione.
> Non so bene perchè ci sono questi continui riallineamenti.

**Comment by RossoDeM** [comment]
> Sarebbe comodo avere un file con solo i giornalieri, ma non è possibile, visto che ogni volta vengono aggionati i dati di molti giorni precedenti.
> Ad esempio nel file del 25 Marzo rispeto al file del 24 Marzo sono state aggiornate le somministrazioni del 4 Gennaio perchè la regione Liguria ha aggiunto 60 somministrazioni prima non dichiarate. Analogamente la regione Lazio ha aggiunto 18 somministrazioni del 6 Gennaio, ancora il Lazio ha aggiunto 10 somministrazioni del 7 Gennaio, etc......
> Le modifiche a ritroso sono tantissime ogni volta e coinvolgono tutto il periodo di vaccinazione.
> Non so bene perchè ci sono questi continui riallineamenti.

Queste continue modifiche sono proprio il frutto della mancanza di dataset giornalieri fin dall'inizio. Se ci fosse stato un monitoraggio quotidiano, non si sarebbe permesso alle regioni di comportarsi in questo modo.
Faccio presente che la Lombardia, ad esempio, il 24 gennaio riportava un totale di:
200,680 dosi ai non sanitari 
17,048 ai sanitari
811 a RSA
46 a Over 80
ma il giorno successivo sono diventate magicamente 114,500 82,098 18,845 e 2,087 rispettivamente.
